README.md

```diff
@@ -158,9 +158,19 @@ The rendered html output will look like this
 </html>
 ```
 
-# Example
+# Examples
 
-To start an example execute `npm run example` and open [http://localhost:8080/index](http://localhost:8080/index).
+## Basic
```

Comment:
Just update the examples here as a list 

+ basic - node examples/basic
+ basic-css ...